docker-hub/api/latest.yaml

```
@@ -521,3 +608,165 @@ paths:
+          examples:
+            application/json:
+              logs:
+                - account: docker
+                  action: repo.tag.push
+                  name: docker/example
+                  actor: docker
+                  data:
+                    digest: 'sha256:c1ae9c435032a276f80220c7d9b40f76266bbe79243d34f9cda30b76fe114dfa'
+                    tag: latest
+                  timestamp: '2021-02-19T01:34:35Z'
+                  action_description: 'pushed the tag latest with the digest sha256:c1ae9c435032a to the repository docker/example'
+        429:
+          description: ''
+          schema: {}
+          examples:
+            application/json:
+              detail: Rate limit exceeded
+              error: false
+        500:
+          description: ''
+          schema: {}
+        default:
+          description: An unexpected error response.
+          schema:
+            $ref: '#/definitions/rpcStatus'
+      parameters:
+        - name: namespace
+          description: Account or namespace to query audit logs for.
+          in: path
+          required: true
+          type: string
+        - name: action
+          description: 'action name one of ["repo.tag.push", ...]. Optional parameter to filter specific audit log actions.'
+          in: query
+          required: false
+          type: string
+        - name: name
+          description: 'name. Optional parameter to filter audit log events to a specific name. For repository events, this is the repository name. For organization events, this is the organization name. For team member events, this is a username.'
+          in: query
+          required: false
+          type: string
+        - name: actor
+          description: actor name. Optional parameter to filter audit log events to the specific user who triggered the event.
+          in: query
+          required: false
+          type: string
+        - name: from
+          description: Start of the time window you wish to query audit events for.
+          in: query
+          required: true
+          type: string
+          format: date-time
+        - name: to
+          description: End of the time window you wish to query audit events for.
+          in: query
+          required: true
+          type: string
+          format: date-time
+        - name: page
+          description: page - specify page number. Page number to get.
+          in: query
+          required: false
+          type: integer
+          format: int32
+          default: '1'
+        - name: page_size
+          description: page_size - specify page size. Number of events to return per page.
+          in: query
+          required: false
+          type: integer
+          format: int32
+          default: '25'
+      tags:
+        - AuditLogs
+  /v2/auditlogs/{namespace}/actions:
+    get:
+      summary: Returns list of audit log actions.
+      description: Get audit log actions.
```

```suggestion
      description: Get audit log actions for reference.
```
Suggestion attempting to clarify the difference between events and actions endpoint. See previous comment. #12554